Configuration/StandardSequences/python/EventInterpretation.py

```diff
@@ -1,3 +1,4 @@
 EventInterpretation = {
-    'top' : 'CommonTools.ParticleFlow.EITopPAG_cff'
+    'top' : 'CommonTools.ParticleFlow.EITopPAG_cff',
+    'MiniAODfromMiniAOD' : 'PhysicsTools.PatAlgos.slimming.MiniAODfromMiniAOD_cff'
```

Comment:
For now, for people to use it, is this fine?
This will need to go into 74X as well, so we should decide what we want quickly.